doc-content/jbpm-docs/src/main/asciidoc/TaskService/TaskServiceAPIs-section.adoc

```diff
@@ -7,15 +7,15 @@ This allows clients  to integrate (at a low level) with the human task service.
 Note that end users should probably not  interact with this low-level API directly, but use one of the more user-friendly task clients (see below)  instead.
 These clients offer a graphical user interface to request task lists, claim and complete tasks,  and manage tasks in general.
 The task clients listed below use the Java API to internally interact with  the human task service.
-Of course, the low-level API is also available so that developers can use it in  their code to interact with the human task service directly. 
+Of course, the low-level API is also available so that developers can use it in  their code to interact with the human task service directly.
```

Comment:
```suggestion
Of course, the low-level API is available so that developers can use it in their code to interact with the human task service directly.
```